### ISSUE TITLE
Adding this gem to Rails application somehow makes Rails scope to "engine"

### ISSUE THREAD

**Issue by vit-panchuk** [codevise/pageflow-embedded-video#18]
If I add gem 'pageflow-embedded-video' to Gemfile of my Rails app I cannot run a server.
Terminal:
```
$ rails s
Error: Command not recognized
Usage: rails COMMAND [ARGS]
The common Rails commands available for engines are:
generate    Generate new code (short-cut alias: "g")
destroy     Undo code generated with "generate" (short-cut alias: "d")
 
All commands can be run with -h for more information.
 
If you want to run any commands that need to be run in context
of the application, like `rails server` or `rails console`,
you should do it from application's directory (typically test/dummy).
exited with code 1
```
Gemfile:
```
...

gem 'pageflow'

# Required for Rails 4.2 compatibility
gem 'state_machine', git: 'https://github.com/codevise/state_machine.git'

gem 'pageflow-react'
gem 'pageflow-before-after'
gem 'pageflow-linkmap-page'
gem 'pageflow-external-links'
gem 'pageflow-text-page'
gem 'pageflow-timeline-page'
gem 'pageflow-embedded-video'

...
```

config/initializers/pageflow.rb:
```
...

config.plugin(Pageflow::TimelinePage.plugin)
config.plugin(Pageflow::LinkmapPage.plugin)
  
config.page_types.register(Pageflow::BuiltInPageType.background_image)
config.page_types.register(Pageflow::BuiltInPageType.background_video)
config.page_types.register(Pageflow::BuiltInPageType.video)
config.page_types.register(Pageflow::BuiltInPageType.audio)
config.page_types.register(Pageflow::BuiltInPageType.audio_loop)
config.page_types.register(Pageflow::BeforeAfter.page_type)
config.page_types.register(Pageflow::ExternalLinks.page_type)
config.page_types.register(Pageflow::TextPage.page_type)
config.page_types.register(Pageflow::EmbeddedVideo.page_type)

...
```

**Comment by tf** [comment]
Hi,

I think this is caused by the fact that the most recent version of this gem incorrectly exposes its bin stubs as executables. I've [fixed that on master](https://github.com/codevise/pageflow-embedded-video/commit/f20d59e1552bf59394ca9cfa6b57b683b78704c6) already and will release a new patch level version later today. You should be able to work around the issue by either using the `bin/rails` bin stub from your project or by running `bundle exec rails` instead of just `rails`.

**Comment by tf** [comment]
I've released 0.3.1. Just to confirm that we are talking about the same problem, can you run `cat $(which rails)` and check whether the last line is `load Gem.activate_bin_path('pageflow-embedded-video', 'rails', version)`. To fix your environment, you either have to reinstall the `railties` gem or manually change the line to `load Gem.activate_bin_path('railties', 'rails', version)`. Installing 0.3.1 of this gem should not cause the same problem anymore.

**Comment by vit-panchuk** [comment]
Thanks, updating to 0.3.1 solved the problem
